roles/edpm_neutron_dhcp/defaults/main.yml

```diff
@@ -71,7 +71,7 @@ edpm_neutron_dhcp_rootwrap_DEFAULT_rlimit_nofile: 1024
 # neutron-dhcp-agent.conf
 # DEFAULT
 edpm_neutron_dhcp_agent_DEFAULT_state_path: '/var/lib/neutron'
-edpm_neutron_dhcp_agent_DEFAULT_host: '{{ ansible_facts["nodename"] }}'
+edpm_neutron_dhcp_agent_DEFAULT_host: '{{ canonical_hostname }}'
```

Comment:
You need to do similar change in the roles/edpm_neutron_sriov/defaults/main.yml too

Reply:
We agreed over slack that the sriov var is unused and will be dropped separately